DigitalLearningSolutions.Web/Views/TrackingSystem/CentreConfiguration/EditRegistrationPrompt.cshtml

```diff
@@ -0,0 +1,112 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem
+@model EditRegistrationPromptViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = "Edit delegate registration prompt";

```

Comment:
Should the title be modified on error as per https://service-manual.nhs.uk/design-system/components/error-summary